.github/scripts/sync-up-dotnet-ver.js

```diff
@@ -0,0 +1,150 @@
+const fse = require("fs-extra");
+const path = require("path");
+const semver = require("semver");
+const targetPkgName = process.argv[2];
+const targetPath = path.join(__dirname, "../../packages", targetPkgName);
+const xml2js = require(path.join(targetPath, "node_modules/xml2js"));
+// ---- target pkg name and version -----
+const pkgName = require(path.join(targetPath, 'package.json')).name;
+const pkgVer = require(path.join(targetPath, "package.json")).version;
+console.log("========= pkg name:", pkgName, " pkg version: ", pkgVer);
+// parse csproj file as XML
+function parseXml(xml) {
+    return new Promise((resolve, reject) => {
+        xml2js.parseString(xml, (err, result) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+}
+
+// update target(simpleauth or function-extension) csproj file
+async function updateCurrentCSprojVer(csprojFile) {
+    const file = await fse.readFile(csprojFile);
+    const result = await parseXml(file);
+    for (let prop of result['Project'].PropertyGroup) {
+        if (prop.Version) {
+            prop.Version[0] = pkgVer;
```

Comment:
Suggest passing in the pkgVer as parameter instead of using it as global parameter. By this way we can make the method self-contain.

Reply:
yes, pass the ver as paremeter